packages/@tailwindcss-upgrade/src/template/prepare-config.ts

```diff
@@ -65,6 +65,7 @@ export async function prepareConfig(
       configFilePath: fullConfigPath,
     }
   } catch (e: any) {
+    console.error(e)
```

Comment:
Do we want to keep this one? We are logging the message below it as well

Reply:
leftover, sorry!